### PR TITLE
Add new cluster DANDI

### PR DESCRIPTION
**Pull request by GeorgianaElena**
PHASE 1 and 2 of https://github.com/2i2c-org/infrastructure/issues/3824

### CODE REVIEW

**Review by yuvipanda — APPROVED** [review]
@GeorgianaElena this lgtm, and I think we should actually merge this if it's been deployed so far - so we can count 'phase 1 and phase 2' as completed.

File: config/clusters/dandi/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: dandi
+provider: aws
```
Comment (consideRatio) [inline config/clusters/dandi/cluster.yaml:2]:
```suggestion
provider: aws # https://2i2c.awsapps.com/start#/
```

File: eksctl/dandi.jsonnet [hunk 2]
```diff
@@ -28,6 +28,15 @@ local notebookNodes = [
     { instanceType: "r5.xlarge" },
     { instanceType: "r5.4xlarge" },
     { instanceType: "r5.16xlarge" },
+    {
+     instanceType: "g4dn.xlarge",
```
Comment (consideRatio) [inline eksctl/dandi.jsonnet:32]: @yuvipanda should dandi have 2xlarge also like the linc hub?

The cost is ~50% higher (0.75 USD/hour compared to 0.53)

Comment (consideRatio) [reply eksctl/dandi.jsonnet:32]: Ahhttps://github.com/2i2c-org/infrastructure/issues/3824#issuecomment-2024299035 declared that to be the case.

Comment (GeorgianaElena) [reply eksctl/dandi.jsonnet]: @consideRatio, I've added https://github.com/2i2c-org/infrastructure/pull/3866/commits/392979ce325791b8c90240d635b656f3c5897cd8 to also support 2xlarge gpu machines. Does that look right to you or should we have _only_ 2xlarge machines?

**Comment by GeorgianaElena** [comment]
With the grafana setup, phase 1 and 2 should now be ready. Merging PR

**Comment by GeorgianaElena** [comment]
thank you @consideRatio and @yuvipanda for the feedback

File: eksctl/dandi.jsonnet (hunk 2 above)
Comment (yuvipanda) [reply eksctl/dandi.jsonnet]: @GeorgianaElena yep, we should have both.

**Pull request merged**